apps/web/components/v2/eventtype/EventLimitsTab.tsx

```diff
@@ -159,149 +159,107 @@ export const EventLimitsTab = (props: Pick<EventTypeSetupInfered, "eventType">)
-                          value={period.type}
-                          className="min-w-4 flex h-4 w-4 cursor-pointer items-center rounded-full border border-black bg-white focus:border-2 focus:outline-none ltr:mr-2 rtl:ml-2">
-                          <RadioGroup.Indicator className="relative flex h-4 w-4 items-center justify-center after:block after:h-2 after:w-2 after:rounded-full after:bg-black" />
-                        </RadioGroup.Item>
-                        {period.prefix ? <span>{period.prefix}&nbsp;</span> : null}
-                        {period.type === "ROLLING" && (
-                          <div className="flex h-9">
-                            <Input
-                              type="number"
-                              className="block w-16 rounded-md border-gray-300 py-3 text-sm [appearance:textfield] ltr:mr-2 rtl:ml-2"
-                              placeholder="30"
-                              {...formMethods.register("periodDays", { valueAsNumber: true })}
-                              defaultValue={eventType.periodDays || 30}
-                            />
-                            <select
-                              id=""
-                              className="block h-9 w-full rounded-md border-gray-300 py-2 pl-3 pr-10 text-sm focus:outline-none"
-                              {...formMethods.register("periodCountCalendarDays")}
-                              defaultValue={eventType.periodCountCalendarDays ? "1" : "0"}>
-                              <option value="1">{t("calendar_days")}</option>
-                              <option value="0">{t("business_days")}</option>
-                            </select>
-                          </div>
-                        )}
-                        {period.type === "RANGE" && (
-                          <div className="inline-flex space-x-2 ltr:ml-2 rtl:mr-2 rtl:space-x-reverse">
-                            <Controller
-                              name="periodDates"
-                              control={formMethods.control}
-                              defaultValue={periodDates}
-                              render={() => (
-                                <DateRangePicker
-                                  startDate={formMethods.getValues("periodDates").startDate}
-                                  endDate={formMethods.getValues("periodDates").endDate}
-                                  onDatesChange={({ startDate, endDate }) => {
-                                    formMethods.setValue("periodDates", {
-                                      startDate,
-                                      endDate,
-                                    });
-                                  }}
-                                />
-                              )}
+      <Controller
+        name="periodType"
+        control={formMethods.control}
+        render={({ field: { value } }) => (
+          <SettingsToggle
```

Comment:
Moves bulk of code into the component - children are passed through to be rendered on `true`